### PR TITLE
Adds Costa Rican and Panamanian payment method brand icons supported by Tilopay

### PR DESCRIPTION
**Pull request by amurillotilopay**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.
- [x] I recognize that if my icon is not approved by the Shopify Partners team it may not receive review nor merger.

Brand icon’s brand guidelines:
- Clave : Not provided by organization owner
- Credix: Not provided by organization owner
- Sinpe Móvil : [Brand guideline book](https://www.bccr.fi.cr/sistema-de-pagos/DocInfoGeneral/Manual_marca_Sinpe_Movil.pdf)

### CODE REVIEW

**Comment by hnannetilo** [comment]
Hi @larouxn , how can we request these new icons to be reviewed?  Just wondering if there's something else that we need to do other than the pull request.  Thank you!

**Comment by amurillotilopay** [comment]
Hi @larouxn  Can you review this PR? Thanks

**Comment by larouxn** [comment]
👋 Hello all, looks like adjustments need to be made.

https://github.com/activemerchant/payment_icons/runs/7738472671?check_suite_focus=true

**Comment by amurillotilopay** [comment]
Thanks @larouxn , the svg now is fixed.

**Review by larouxn — APPROVED** [review]
The Clave icon has an extra black border but I guess that's intentional?

Looks good to me :tophat: 
image

File: app/assets/images/payment_icons/clave_telered.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...ave_telered-path32" fill="#fff" d="M17.22 9.03l.4.28.34-.38c0-.48-.76-.27-.76-.86 0-.34.25-.5.53-.5l.35.12-.06.21H18l-.3-.22-.31.3c0 .48.77.29.77.84 0 .37-.27.6-.6.6l-.39-.14.05-.24h.02"/><path id="pi-clave_telered-path34" fill="#fff" d="M19.84 7.6h.21v1.79h-.21V7.6"/><path id="pi-clave_telered-path36" fill="#fff" d="M21.82 9.03l.4.28.34-.38c0-.48-.76-.27-.76-.86 0-.34.25-.5.54-.5l.34.12-.07.21h-.03l-.29-.22-.32.3c0 .48.77.29.77.84 0 .37-.27.6-.6.6l-.39-.14.04-.24h.03"/><path id="pi-clave_telered-path38" fill="#fff" d="M25.01 9.4h-.21V7.73c-.25 0-.4.02-.54.04v-.16h1.26v.16l-.51-.04V9.4"/><path id="pi-clave_telered-path40" fill="#fff" d="M27.35 8.38h.6v.13h-.6v.75h.44l.23-.03-.02.07.02.09h-.89V7.61H28v.16l-.21-.04h-.44v.65"/><path id="pi-clave_telered-path42" fill="#fff" d="M31.34 7.6c.06.64.12 1.28.19 1.79h-.23c-.04-.52-.08-1.09-.12-1.35-.29.53-.5.96-.65 1.32h-.02l-.14-.3c-.16-.34-.32-.68-.49-1.01-.05.51-.12 1.02-.12 1.35h-.17c.07-.42.21-1.51.24-1.79h.06c.19.42.42.9.7 1.37.22-.45.45-.91.7-1.37h.05"/><path id="pi-clave_telered-path44" fill="#fff" d="M33.78 7.91l-.27.67h.54l-.27-.67zm.82 1.48h-.26c-.07-.22-.16-.46-.25-.71h-.62c-.09.23-.18.46-.24.71h-.17l.75-1.82h.06l.74 1.82"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/clave_telered.svg:1]: Can we remove all unused `id` attributes such as `pi-clave_telered-path40`, `pi-clave_telered-path36`?

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
Looks good. Just left one small nitpick comments.

**Comment by amurillotilopay** [comment]
Thanks guys.

@larouxn the extra black border is part of the [logo](https://www.telered.com.pa/en/sistemaclave/eclave/).
image

@tauthomas01 The unused id attributes were removed.

File: app/assets/images/payment_icons/clave_telered.svg (hunk 1 above)
Comment (amurillotilopay) [reply app/assets/images/payment_icons/clave_telered.svg]: @tauthomas01 The unused id attributes were removed.

**Review by tauthomas01 — APPROVED** [review]

**Comment by larouxn** [comment]
Aiming to have these icons live within the week. 🙇‍♂️

**Comment by larouxn** [comment]
👋 Hello, your new/updated icon(s) should now be live everywhere. ✨

**Pull request merged**